hack/llmo-demo/worker-plane/llm-operator-values.yaml

```diff
@@ -22,22 +22,24 @@ inference-manager-engine:
   resources:
     limits:
       nvidia.com/gpu: 4
-  autoscaling:
-    enableKeda: false
+
+  # Max concurrent request per model
+  maxConcurrentRequests: 1
+
   preloadedModelIds:
-  - meta-llama-Meta-Llama-3.1-70B-Instruct-q4
-  - deepseek-ai-deepseek-coder-6.7b-base-q4
+  - meta-llama-Meta-Llama-3.1-70B-Instruct-q2_k
   - deepseek-ai-deepseek-coder-6.7b-base
 
+  modelContextLengths:
+    meta-llama-Meta-Llama-3.1-70B-Instruct-q2_k: 16384
```

Comment:
just want to confirm, we want 16K context length, not 64K for this one?

also, I think context length depends on base model, so probably we don't have to include quantization part in model name here

Reply:
> just want to confirm, we want 16K context length, not 64K for this one?

Yes, that's correct. 64K was too large to fit in A10.

> also, I think context length depends on base model, so probably we don't have to include quantization part in model name here

Actually it's better to include as the required memory for the context length also depends on the model parameter size (e.g., 16-bit, 8-bit, 4-bit).